projects/plugins/boost/app/assets/src/js/pages/settings/elements/CriticalCssStatus.svelte

```diff
@@ -38,7 +38,7 @@
 				{#if $criticalCssState.updated}
 					<TimeAgo time={new Date( $criticalCssState.updated * 1000 )} />.
 				{/if}
-				{#if ! $cloudCssAvailable}
+				{#if ! cloudCssAvailable}
```

Comment:
This can probably be inlined since it looks like the variable isn't used anywhere else